app/auto-updater.js

```diff
@@ -1,50 +1,45 @@
-const {autoUpdater} = require('electron');
+const {autoUpdater, app} = require('electron');
 const ms = require('ms');
+const isDev = require('electron-is-dev');
 
 const notify = require('./notify'); // eslint-disable-line no-unused-vars
 const {version} = require('./package');
 
 const {platform} = process;
 const FEED_URL = `https://releases.hyper.is/update/${platform}`;
 
-let isInit = false;
-
-function init() {
-  autoUpdater.on('error', (err, msg) => {
-    console.error('Error fetching updates', msg + ' (' + err.stack + ')');
-  });
-
-  autoUpdater.setFeedURL(`${FEED_URL}/${version}`);
-
-  setTimeout(() => {
-    autoUpdater.checkForUpdates();
```

Comment:
It is better to keep this first call.
We can't wait 30mn after start to notify user that an update is available. If user uses Hyper 15mn per day, he'll never see updates.